Configure

```diff
@@ -773,7 +773,7 @@ while (@argvcopy)
                         next;
                         }
                 }
-        if (/^no-(.+)$/ || /^disable-(.+)$/)
+        if (/^-?no-(.+)$/ || /^-?disable-(.+)$/)
```

Comment:
What about converting this to `/^-?(no|disable)-(.+)$/` to match the previous check?  I think it's clearer.

Reply:
It may be clearer but it will cause either use non-grouping parentheses or replacing $1 with $2

Reply:
Or you use non-capturing groups... 

`/^-?(?:no|disable)-(.+)$/`